api/ProgressEvent.json

```diff
@@ -70,10 +70,10 @@
               "version_added": "14"
             },
             "firefox": {
-              "version_added": "22"
+              "version_added": "18"
```

Comment:
Confirmed by testing `new ProgressEvent('bla')` in Firefox 18. I'm not 100% confident in our constructor tests, since they don't pass in the required arguments to ensure that the constructor actually succeeds.